clients.go

```diff
@@ -216,3 +239,40 @@ func headersToHTTPHeaders(h *headersList) http.Header {
 	}
 	return headers
 }
+
+func executeLuaScript(l *lua.LState, path string, method string, body string) (string, string, string, error) {
```

Comment:
Just a thought: maybe, users would like to change not only path, but other parts of URL?